CHANGELOG.md

```diff
@@ -45,8 +45,9 @@
 
 - [#907](https://github.com/oauth2-proxy/oauth2-proxy/pull/907) Introduce alpha configuration option to enable testing of structured configuration (@JoelSpeed)
 - [#938](https://github.com/oauth2-proxy/oauth2-proxy/pull/938) Cleanup missed provider renaming refactor methods (@NickMeves)
-- [#925](https://github.com/oauth2-proxy/oauth2-proxy/pull/925) Fix basic auth legacy header conversion (@JoelSpeed)
-- [#916](https://github.com/oauth2-proxy/oauth2-proxy/pull/916) Add AlphaOptions struct to prepare for alpha config loading (@JoelSpeed)
+- [#928](https://github.com/oauth2-proxy/oauth2-proxy/pull/928) Allow * in the email-domain list. (@morarucostel)
+- [#925](https://github.com/oauth2-proxy/oauth2-rpoxy/pull/925) Fix basic auth legacy header conversion (@JoelSpeed)
```

Comment:
It looks like this typo got cleaned up in a PR since yours was made. When you rebase off the latest master can you make sure the link typo stays fixed?